app/helpers/cms_helper.rb

```diff
@@ -192,4 +192,10 @@ def get_resource(fragment_text, fragment_link, button_text, button_class)
       false
     end
   end
+
+  def cms_fragment_content_datetime(identifier, page)
+    # Might not be able to find that particular tag and identifier combo -  hence the try
+    page.fragments.find_by(tag: 'date_not_null', identifier: identifier).try(:datetime)
```

Comment:
@stanleypliu I think you forgot to assign this value to the `date` var here :)

Reply:
Isn't it throwing an exception for you?

Reply:
😓